unit-tests/shared/src/test/scala/javalib/net/URITest.scala

```diff
@@ -237,4 +237,33 @@ class URITest {
     }
   }
 
+  @Test def normalize(): Unit = {
+    def testNormalize(relative: Boolean): Unit = {
+      val first = if (relative) "" else "/"
+      assertEquals(new URI(s"${first}a/b").normalize(), new URI(s"${first}a/b"))
```

Comment:
Please make the expected result be present on the left side of `assertEquals`, and the actual value on the right side. That's an assumption taken by JUnit, otherwise it would lead to confusing messages in the logs in case of the failed tests.